content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1362 @@
+
+![Arduino Linux Wizard Wi-Fi® connection successful](assets/OOTB_wifi_connected.png "Arduino Linux Wizard Wi-Fi® connection successful")
+
+Now, you can click **OK** and be redirected to the Arduino Linux Wizard homepage below.
+
+![Arduino Linux Wizard Homepage](assets/OOTB_homepage.png "Arduino Linux Wizard Homepage")
+
+***You can change your network by clicking on the Settings button and repeat the steps above.***
+
+#### Arduino Linux Wizard Homepage
+
+The Arduino Linux Wizard, hosted on the Portenta X8, allows a user to leverage the following capabilities:
+
+- Get board details
+- [Configure Portenta X8 Wi-Fi®](#wi-fi-configuration)
+- [Interact with the board through the embedded Python® Alpine Shell](#portenta-x8-with-python-alpine-shell)
+- [Provision your device to Arduino Cloud](#portenta-x8-with-arduino-cloud)
+- Manage the Linux distribution with the dedicated [Portenta X8 Board Manager](#portenta-x8-board-manager)
+- [Update the board's operating system](#portenta-x8-os-image-update)
+
+#### Portenta X8 with Python Alpine Shell
+
+Click the **Shell** button to start using your Portenta X8 with Python-Alpine.
+
+![Arduino Linux Wizard Shell button](assets/OOTB_homepage_shell.png "Arduino Linux Wizard Shell button")
+
+This shell is running in a Python-Alpine container embedded in Portenta X8. You will find multiple examples under the directory `/root/examples` in this shell. Additionally, you can either add your own package through the command `apk add <packagename>` or start exploring the packages available online at [this link]( https://pkgs.alpinelinux.org/packages).
+
+![Arduino Linux Wizard Python-Alpine Shell](assets/OOTB_alpine_shell.png "Arduino Linux Wizard Python-Alpine Shell")
+
+## Portenta X8 OS Image Update
+
+It is recommended that you check every now and then to see if your Portenta X8 image version is up to date to have the latest security updates.
+
+***To review the image version history, refer to the [__Portenta X8 firmware release notes__](https://docs.arduino.cc/tutorials/portenta-x8/x8-firmware-release-notes/).***
+
+There are four ways to update your Portenta X8 are described:
+
+* [Update for OS release V.399](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing/#update-for-os-release-v399)
+* [Update through Arduino Linux Wizard experience](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing/#update-through-arduino-linux-wizard-experience)
+* [Update through Portenta X8 Manager in your Arduino Cloud for Business account (available for all OS releases)](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing/#update-with-portenta-x8-board-manager)
+* [Update using the `uuu` tool (compatible with custom images)](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing/#update-using-uuu-tool)
+
+You can find more details in the dedicated tutorial here: [**How To Flash Your Portenta X8**](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing).
```

Comment:
Please rename the tutorial "How to Flash Your Portenta X8" with "How To Update Your Portenta X8"

Reply:
Updated!